server/lib/alert-manager.js

```diff
@@ -45,16 +46,24 @@ ${$t('server.email.signature')}
 
     async checkAlertsForOperations(operations) {
         try {
+            // Default currency
+            let defaultCurrency = await Config.byName('defaultCurrency').value;
             // Map account to names
             let accounts = await Account.all();
-            let accountNames = new Map;
+            let accountsMap = new Map;
             for (let a of accounts) {
-                accountNames.set(a.accountNumber, a.title);
+                accountsMap.set(a.accountNumber, {
+                    title: a.title,
+                    currency: a.currency || defaultCurrency
```

Comment:
how about directly storing the currency format function here instead? So that you can pass it to formatOperationMessage and there's no need to call makeFormat for each alert.